src/api/application/application.ts

```diff
@@ -440,7 +441,16 @@ export class Application extends EmitterBase<ApplicationEvents> {
         return this.wire.sendAction('get-shortcuts', this.identity)
             .then(({ payload }) => payload.data);
     }
-
+    /**
+    * Retrieves current application's views.
+    * @experimental
+    * @return {Promise.Array.<BrowserView>}
+    * @tutorial Application.getViews
+    */
+    public async getViews(): Promise<Array<BrowserView>> {
```

Comment:
not sure it's a good idea, but I'll say it - due to the impermanent nature of Views' target, wouldn't getCurrentViews be a better name choice?

Reply:
oh right